tests/applications/api.test.ts

```diff
@@ -62,6 +102,31 @@ describe('Applications API', () => {
         scope.done();
     });
 
+    it('List Application Installations', async () => {
```

Comment:
@AksBad007 I just noticed that you don't do any assertions in these tests (using the `expect` function) like in other tests. Could you please improve this with a separate PR?

Reply:
Sure, I'm on it.